src/mqtt_client.rs

```diff
@@ -484,6 +473,7 @@ where
                             // Note(unwrap): The vector is guaranteed to be as large as the largest MQTT
                             // message size, so the properties (which are a portion of the message) will
                             // always fit into it.
+                            // TODO: Figure out how to handle cached properties for listing?
```

Comment:
Current thought is that we just don't cache the properties anymore. Instead, we could only cache:
* The response topic
* The correlation data

Still not entirely sure how to properly cache this data, and I'm not sure if it's required. Perhaps we could just entirely omit the properties when republishing. Alternatively, the user could provide yet another buffer to miniconf to store this cached data into?

Reply:
I have opted to drop the caching of properties, so CD isn't maintained through all the listed elements - just the initial reply to the list request.